ui/src/components/Investor/InvestorTrade.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../index.scss";
+@import "./OrderForm.scss";
+
+@mixin investor-trade {
+    .investor-trade {
+        height: 100%;
+
+        .order {
+            h3 {
+                display: flex;
+                align-items: center;
+            }
+
+            .candlestick-icon {
+                margin-right: $spacing-s;
+            }
+            .order-form {
+                @include order-form;
+                flex: 1;
+                margin-right: $spacing-m;
+            }
+        }
+    }
+}
+
+@include investor-trade;
```

Comment:
Why @include this mixin here?

Reply:
possible recursive implications nature make me nervous

Reply:
initially, I just wanted to include it in the same file so the mixin would be used. I'll move this to the `Investor.scss` that you created after I rebase on the new changes